rollup.config.js

```diff
@@ -29,7 +30,7 @@ const banner = `/*!
 const plugins = [
   json(),
   sass({
-    output: "dist/introjs.css",
+    output: `${outputPath}/minified/introjs.min.css`,
```

Comment:
Why do we need to keep the minified files in a separate folder?

Reply:
to match the existing package structure, see this https://www.jsdelivr.com/package/npm/intro.js